src/lib/people/team/team.ts

```diff
@@ -0,0 +1,47 @@
+import * as v from 'valibot';
+
+export const committees = [
+    'Executive',
+    'Innovation',
+    'Service',
+    'External Relations',
+    'Membership & Internals',
+    'Branding & Creatives',
+    'Engineering',
+];
+
+const MemberSchema = v.object({
```

Comment:
Typically, I put these models inside a `$lib/models` or `$lib/server/models` module to keep the data model separate from the app logic. Personally, I prefer putting it in `$lib/server/models` by default and then moving to `$lib/models` when the browser bundle requires it.